emva1288/process/data.py

```diff
@@ -275,24 +277,46 @@ def _get_spatial_data(self, d, postfix=''):
         are:
                - *'sum'*: the sum image preserved from input,
                - *'L'*: the number of image summed,
+               - *'M'*: the number of columns for one image
+               - *'N'*: the number of rows for one image
                - *'avg_var'*: variance of the the average image
                - *'avg_mean'*: mean of the the average image
                - *'var_mean'*: the variance of the variance image
+               - *'avg_var_cav'*: residual column spatial variance
+               - *'avg_var_rav'*: residual row spatial variance
         """
 
         # This cast is just in case the original images are unint
-        sum_ = d['sum'].astype(np.int64)
+        sum_ = d['sum'].astype(np.int64)  # <y> --> eqn(33)
         pvar_ = d['pvar'].astype(np.int64)
 
         L = d['L']
-        avg_ = sum_ / (1.0 * L)
+        M = self.cols
+        N = self.rows
```

Comment:
The standard specifies that M x N images have "M rows and N columns."

Reply:
my bad, I was confused at first. I thought this is a naming error, but now I see this is a bug.